app/src/main/java/com/efedaniel/spotifystats/ui/proton/components/chips/ProtonChips.kt

```diff
@@ -0,0 +1,24 @@
+package com.efedaniel.spotifystats.ui.proton.components.chips
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.material3.SuggestionChip
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.efedaniel.spotifystats.ui.proton.tokens.color.ProtonColor
+
+@Composable
+fun Chip(genre: String) {
```

Comment:
I noticed we have a file for other composable in the project so I followed the same pattern

Reply:
I see. Good observation. 
Those are components in the design system i.e. Proton. So we have like ProtonText, ProtonButton, Proton Snackbar, etc.

It will be a good idea to have ProtonChip but that will mean it has to be generic. i.e. 
- it will need to have `text` as the parameter instead of `genre`
- It will need to hoist the onClick parameter
- And it would need to either have a standardized color or have a parameter for color.

Reply:
True, I will adjust that and make the Proton chip generic so that it can be reusable

Reply:
I have a question, why is there a prefix "Proton" before the real name of the composables